charts/paperless/README.md

```diff
@@ -96,10 +96,11 @@ The following values can be used to adjust the helm chart.
 | rbac.labels | object | `{}` | Additional labels for the role and role binding objects. |
 | resources | object | `{}` | Compute resources used by the container. More info [here](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/). |
 | secret.annotations | object | `{}` | Additional annotations for the secret object. |
-| secret.create | bool | `true` | Create a new secret containing the [secret key](https://paperless-ngx.readthedocs.io/en/latest/configuration.html#hosting-security). |
-| secret.existingSecret | string | `""` | Use an existing secret to store the [secret key](https://paperless-ngx.readthedocs.io/en/latest/configuration.html#hosting-security). |
+| secret.create | bool | `true` | Create a new secret containing the [secret values](https://paperless-ngx.readthedocs.io/en/latest/configuration.html#hosting-security). |
+| secret.existingSecret | string | `""` | Use an existing secret to store the [secret values](https://paperless-ngx.readthedocs.io/en/latest/configuration.html#hosting-security). Please note: keys inside the existing secret must match the keys from below! |
 | secret.labels | object | `{}` | Additional labels for the secret object. |
-| secret.secretKey | string | `""` | [Secret key](https://paperless-ngx.readthedocs.io/en/latest/configuration.html#hosting-security) used when not using an existing secret. |
+| secret.values | list | `[{"name":"PAPERLESS_SECRET_KEY","value":"{{ randAlphaNum 42 | b64enc }}"}]` | Secret values used when not using an existing secret. |
```

Comment:
Misleading description to the overlying `.Values.secret.values` object. Probably need to add/change the code comment to optimize this...